dataplane/standalone/sai/hostif.cc

```diff
@@ -68,11 +66,15 @@ lemming::dataplane::sai::CreateHostifRequest convert_create_hostif(
         msg.set_obj_id(attr_list[i].value.oid);
         break;
       case SAI_HOSTIF_ATTR_NAME: {
-        std::ostringstream s;
-        s << "ip link set eth" << nextIdx++ << " name "
-          << attr_list[i].value.chardata;
-        LOG(INFO) << s.str();
-        system(s.str().c_str());
+        int idx;
```

Comment:
If short-term then add a TODO on the long-term fix?

Reply:
https://github.com/openconfig/lemming/blob/main/dataplane/standalone/apigen/ccgen/ccgen.go#L161